airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/message/DestinationMessage.kt

```diff
@@ -503,7 +505,7 @@ class DestinationMessageFactory(
         val descriptor = streamState.streamDescriptor
         return Checkpoint(
             stream = DestinationStream.Descriptor(descriptor.namespace, descriptor.name),
-            state = streamState.streamState
+            state = runCatching { streamState.streamState }.getOrNull()
```

Comment:
just curious, why do we need the runCatching? is `.streamState` throwing an NPE or something?

Reply:
We don't. I wasn't sure at first where the exception was happening. `streamState`'s signature is a non-null JsonNode. I left it in to be pointlessly cautious